lib/arangoex/simple.ex

```diff
@@ -1,28 +1,76 @@
 defmodule Arangoex.Simple do
-  @moduledoc false
+  @moduledoc """
+  ArangoDB HTTP Interface for Simple Queries
 
-  alias Arangoex.JSON
+  ## TODO
+
+  * Implement function for `PUT /_api/simple/within-rectangle`
+  * Specify typespecs
+
+  """
 
   use Arangoex, base_url: ["/", "_api", "/", "simple"]
+  alias Arangoex.{JSON,Response}
```

Comment:
Ebert made this comment.  It's cool, I like the shorter multi-alias syntax better.